src/terrafort/templates/aws_security_group_rule.tf

```diff
@@ -1,25 +1,23 @@
-{% for range in resource.IpRanges %}
+{% for range in resource.IpRanges -%}
 {% set rule_num = count() %}
 resource "aws_security_group_rule" "{{ resource.name }}{% if rule_num > 0 %}_{{ rule_num }}{% endif %}" {
-  security_group_id = "{{ resource.id }}"
+  security_group_id = {{ resource.id }}
   type              = "{{ resource.type }}"
```

Comment:
will the quotes be an issue with `resource.type` as well?

Reply:
no, this field requires quotes